Source/Editor/Gizmo/TransformGizmoBase.Selection.cs

```diff
@@ -56,104 +56,107 @@ private void SelectAxis()
             _activeAxis = Axis.None;
             switch (_activeMode)
             {
-            case Mode.Translate:
-            {
-                // Axis boxes collision
-                if (XAxisBox.Intersects(ref localRay, out intersection) && intersection < closestintersection)
-                {
-                    _activeAxis = Axis.X;
-                    closestintersection = intersection;
-                }
-                if (YAxisBox.Intersects(ref localRay, out intersection) && intersection < closestintersection)
-                {
-                    _activeAxis = Axis.Y;
-                    closestintersection = intersection;
-                }
-                if (ZAxisBox.Intersects(ref localRay, out intersection) && intersection < closestintersection)
+                case Mode.Translate:
```

Comment:
I think the "missing" indent for the `case` labels is intentional.

Reply:
Yeah we don't indent cases in switches.